packages/nimble-components/src/table-column/icon/index.ts

```diff
@@ -20,6 +19,10 @@ import type { MappingConfig } from '../enum-base/models/mapping-config';
 import { MappingIconConfig } from '../enum-base/models/mapping-icon-config';
 import { MappingSpinnerConfig } from '../enum-base/models/mapping-spinner-config';
 
+const cellPadding = 8;
```

Comment:
That's a good question. I did it this way since that was the existing pattern for defining the default minimum column width:
https://github.com/ni/nimble/blob/2c782847534b9af993d7e97e41d773f150b60793/packages/nimble-components/src/table-column/base/types.ts#L47-L60

I think the options are:
- leave this as-is
- add comments that these are the default values for `mediumPadding` and `iconSize`
- export the values from `table-column/base/types` (maybe with added comments there?) so that we are limiting the places we reference the default token values, perhaps with some values renamed to be more generic (e.g. `iconSize` rather than having both `groupIconSize` and `sortIconSize`).

Thoughts on a preferred approach? I'm leaning towards exporting the values from the existing shared `types` file. I originally had shied away from that because I didn't want to make this PR too big, but I don't think that change would actually add too much extra code.

Reply:
I think I also like the exporting idea. I'm not too worried about PR size. It was pretty easy to get through.